Staging_Dev/20190215T061300Z_remove_temp_stronghold_teleport.sql

```diff
@@ -0,0 +1,9 @@
+USE [perpetuumsa]
+GO
+
+delete from dbo.teleportdescriptions
+where id = 403
```

Comment:
Please replace the ID lookup with name columns to find the correct row independent of auto incremented ids.

Reply:
The table does not contain a column for the name, but I used description column instead.